src/Education/Commands/RuleHelpUserControl.xaml.cs

```diff
@@ -21,24 +21,35 @@
 using System.Windows.Controls;
 using System.Windows.Navigation;
 using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.Integration;
 
 namespace SonarLint.VisualStudio.Education.Commands
 {
     public sealed partial class RuleHelpUserControl : UserControl
     {
         private readonly IBrowserService browserService;
         private readonly IEducation education;
+        private readonly ILogger logger;
 
-        internal RuleHelpUserControl(IBrowserService browserService, IEducation education)
+        internal RuleHelpUserControl(IBrowserService browserService, IEducation education, ILogger logger)
         {
             this.browserService = browserService;
             this.education = education;
+            this.logger = logger;
 
             InitializeComponent();
         }
 
         public void HandleRequestNavigate(object sender, RequestNavigateEventArgs e)
         {
+            if (!e.Uri.IsAbsoluteUri)
+            {
+                logger.LogVerbose($"[RuleHelpUserControl] Failed to navigate to Uri as it is a relative path: {e.Uri}");
+                logger.WriteLine($"[RuleHelpUserControl] Failed to open URI: {e.Uri}");
```

Comment:
`RuleHelpUserControl` is a bit low-level for "normal" verbosity messages. We tend to use a higher-level description of the feature e.g. `[Education]`